app/js/lib/foxbox/settings.js

```diff
@@ -145,9 +141,14 @@ export default class Settings extends Model {
   }
 
   set pollingEnabled(value) {
-    value = !!value;
```

Comment:
I haven't found any place where we pass something non-boolean here and basically we should not.